pkg/cloudclient/aws/private.go

```diff
@@ -20,7 +20,8 @@ import (
 )
 
 var (
-	instanceType  string = "t2.micro"
+	// NOTE a "nitro" EC2 instance type is required to be used
+	instanceType  string = "t3.micro"
```

Comment:
Just a though; I wonder if there's a way to see if an instance type is "nitro" or not from the AWS API. Then we could make the instance type configurable as long as its "nitro"

Reply:
There looks to be a `describe instance type` API that we could utilize to check whether the provided instance type is indeed nitro by reviewing the instance type's `hypervisor` attribute: https://docs.aws.amazon.com/cli/latest/reference/ec2/describe-instance-types.html

Is this worth putting together an OSD card for?

Reply:
https://issues.redhat.com/browse/OSD-9345